src/NormalizedCollection/libs/Record.js

```diff
@@ -94,16 +94,23 @@ util.inherits(Record, AbstractRecord, {
    * @returns {Object}
    */
   mergeData: function(snaps, isExport) {
-    var map = this.getFieldMap();
-    var data = util.extend.apply(null, util.map(snaps, function(ss) {
-      return map.extractData(ss, isExport);
-    }));
-    if( isExport && snaps.length > 0 && snaps[0].getPriority() !== null ) {
-      if( !util.isObject(data) ) {
-        data = {'.value': data};
+    var data = null, map = this.getFieldMap();
+
```

Comment:
This fixes an error where we incorrectly returned an object with `{.value, .priority}` if the final value was null and `exportVal()` was called. That doesn't match the Firebase SDK (null values have no priority) and Firebase just returns `null`.